pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "Heksher"
+version = "0.1.0"
+description = "Heksker"
+authors = ["Biocatch LTD"]
```

Comment:
I think authors have to be in the format of `"NAME <MAIL>"` 
https://python-poetry.org/docs/pyproject/#authors

Reply:
You can put serverteam@biocatch.com I guess